src/components/manage/ManageLookup.js

```diff
@@ -0,0 +1,21 @@
+import React, { Component } from "react";
+import { Text, View } from "react-native";
+import Constants from "../../constants/PortfolioConstants";
+import LookupNavigator from "../../navigation/LookupNavigator"
+
+class ManageLookup extends React.Component {
+  static navigationOptions = {
+    title: Constants.MANAGE_LOOKUP_TITLE,
+    headerTintColor: Constants.FORE_COLOR_CODE,
+    headerTitleStyle: {
+      fontWeight: "300",
```

Comment:
We should move font-family, font-size and font-weight etc to constants like font-base, font-large, font-x-large etc. That way we get consistency across app and easy to upgrade ability...